README.md

```diff
@@ -1,4 +1,4 @@
-# Node.js Internationalization
+# Node.js Internationalization [![Crowdin](https://badges.crowdin.net/nodejs/localized.svg)](https://crowdin.com/project/nodejs)
```

Comment:
It'd suggest keeping it separate from the H1, may mess with the auto anchor
```suggestion
# Node.js Internationalization

[![Crowdin](https://badges.crowdin.net/nodejs/localized.svg)](https://crowdin.com/project/nodejs)
```

Reply:
Reasonable!